src/daemon/scheduler.rs

```diff
@@ -146,20 +149,21 @@ impl Scheduler {
 
         debug!("Adding tasks at start up");
 
-        let use_jitter = cas.len() >= SCHEDULER_USE_JITTER_CAS_THRESHOLD;
+        let too_many_cas = cas.len() >= SCHEDULER_USE_JITTER_CAS_THRESHOLD;
```

Comment:
I find these "too_many" variable names confusing and the first one is about jitter and doesn't mention that. Maybe rename these flags to `use_jitter` and `do_repo_resync` or something?

Reply:
Oh it was called `use_jitter` before...

Reply:
improved, I think..